hq/app/controllers/SnykController.scala

```diff
@@ -31,18 +32,22 @@ class SnykController(val config: Configuration, val configraun: com.gu.configrau
     Action.async {
       attempt {
         for {
+
           token <- getSnykToken
           requiredOrganisation <- getSnykOrganisation
 
           organisationResponse <- Snyk.getSnykOrganisations(token, wsClient)
-          organisation <- SnykDisplay.getOrganisation(organisationResponse.body, requiredOrganisation)
+          organisations <- SnykDisplay.parseOrganisations(organisationResponse.body, requiredOrganisation)
```

Comment:
WSClient responses have a `.json` method that returns the body as a JsValue, which might save you some hassle?

Reply:
Decided to stick with parsing the json body and coping with non-json data, just in case.  It would change very little code now.